src/components/pages/blog/posts-board/posts-board.jsx

```diff
@@ -36,10 +37,12 @@ const PostsBoard = ({ categories, posts, currentCategory, currentPage, numPages
           numPages={numPages}
           currentCategory={currentCategory}
         />
+        <CommunityLinks />
```

Comment:
The community block looks great, but it's better to have it in `/templates/blog` to avoid that case when `Container` is inside another `Container`

![image](https://user-images.githubusercontent.com/48465000/163546830-1fd813b9-aa23-48d6-bb60-bfdb8f9e4b4d.png)

![image](https://user-images.githubusercontent.com/48465000/163546390-aa05edf7-5dd0-4dd4-9eb8-b068f43ed865.png)